src/core_plugins/spy_modes/public/table_spy_mode.js

```diff
@@ -1,4 +1,4 @@
-import 'ui/agg_table';
+// import 'ui/agg_table';
```

Comment:
Why comment this out instead of removing it?

Reply:
that was an oversight, I'll clean that up.